www/src/components/utils/InstallDropdownButton.tsx

```diff
@@ -8,9 +8,35 @@ import {
 import { Link } from 'react-router-dom'
 import capitalize from 'lodash/capitalize'
 
-import RepositoryContext from '../../contexts/RepositoryContext'
+export interface Recipe {
```

Comment:
Since the GraphQL generated types were merged already we can use them instead of creating duplicated types (src/generated/graphql.tsx).

Reply:
This is only a common pary of repository and stack recipes (data that is required to render the component). In this case I would like to keep it minimal instead of using generated type.